pyvista/utilities/reader.py

```diff
@@ -980,6 +983,185 @@ class Plot3DMetaReader(BaseReader):
     _class_reader = staticmethod(_vtk.lazy_vtkPlot3DMetaReader)
 
 
+class CGNSReader(BaseReader):
```

Comment:
It seems at first glance thatthis reader supports all the functionality from `PointCellDataSelection`

Reply:
You're absolutely right, and I should have looked again at your reader implementation. Resolved in https://github.com/pyvista/pyvista/pull/2366/commits/893bb803cf5a868e540cfce2d8c87c9198e47500.